packages/medusa-react/src/hooks/admin/collections/mutations.ts

```diff
@@ -62,3 +64,60 @@ export const useAdminDeleteCollection = (
     )
   )
 }
+
+
+/**
+ * Hook returns functions for addition of multiple products to a collection.
```

Comment:
todo: Minor fix to grammar
```suggestion
 * Hook returns function for adding multiple products to a collection.
```